vortexasdk/endpoints/fleet_utilisation_avg_distance_timeseries.py

```diff
@@ -199,7 +199,7 @@ def search(
             "filter_ship_to_ship": sts_filter["x_filter"],
             # if charterer toggle is True, apply cross filter
             # else make it false
-            "filter_charterer_exists": filter_charterer_exists == True,
```

Comment:
Is this a recommendation from the linter? If yes, this is dangerous in projects that include `numpy`. `numpy` (especially older versions) go hard on reimplementing boolean. 1.20 onwards one of those implementations is at least an alias for python's bool, and gives you a deprecation warning, but the other implementation doesn't.
<img width="252" alt="Screenshot 2022-09-01 at 15 58 27" src="https://user-images.githubusercontent.com/503380/187946430-1f64eda6-5930-475a-8e1e-736e4e36a5eb.png">
Raising this, because this has caused real bugs at Vortexa in the past.

Reply:
This was a recommendation from Flake8. I wasn't aware of this issue with Numpy. I'll just revert the change and ignore this error code. Thanks for raising.